### PR TITLE
Add CleanAllNetworksByTargetNamespace function to sriov package

### PR DESCRIPTION
**Pull request by mcornea**
Add function which deletes all sriov networks from the operator namespace which match a specific NetworkNamespace spec.

### CODE REVIEW

File: pkg/sriov/network.go [hunk 1]
```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// CleanAllNetworksByTargetNamespace deletes all networks matched by their NetworkNamespace spec.
+func CleanAllNetworksByTargetNamespace(
+	apiClient *clients.Settings,
+	operatornsname string,
```
Comment (kononovn) [inline pkg/sriov/network.go:425]: Please check that parameter is not empty

File: pkg/sriov/network.go [hunk 2]
```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
...
+	targetnsname string,
```
Comment (kononovn) [inline pkg/sriov/network.go:426]: Please check that parameter is not empty

File: pkg/sriov/network.go [hunk 3]
```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
...
+	options metaV1.ListOptions) error {
+	networks, err := List(apiClient, operatornsname, options)
+
```
Comment (kononovn) [inline pkg/sriov/network.go:429]: Please add glog.V(100).Infof with log what this function is going to perform

**Review by kononovn — CHANGES_REQUESTED** [review]
Please fix few comments

File: pkg/sriov/network.go [hunk 4]
```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
...
+	if err != nil {
+		glog.V(100).Infof("Failed to list sriov networks: %s", operatornsname)
```
Comment (trewest) [inline pkg/sriov/network.go:431]: nit: "Failed to list sriov networks in namespace %s", operatornsname

File: pkg/sriov/network.go (hunk 1 above)
Comment (mcornea) [reply pkg/sriov/network.go:425]: Added check.

File: pkg/sriov/network.go (hunk 2 above)
Comment (mcornea) [reply pkg/sriov/network.go:426]: Added check.

File: pkg/sriov/network.go (hunk 3 above)
Comment (mcornea) [reply pkg/sriov/network.go:444]: Added.

File: pkg/sriov/network.go (hunk 4 above)
Comment (mcornea) [reply pkg/sriov/network.go]: Updated.

**Review by kononovn — APPROVED** [review]
lgtm

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**